resources/views/account/listing.blade.php

```diff
@@ -10,7 +10,7 @@
             <div v-if="wishlists" class="space-y-0.5 my-5">
                 <div
                     v-for="(wishlist, index) in wishlists"
-                    class="bg-ct-inactive-100 p-6 sm:px-9 sm:pt-6 sm:pb-9 rounded flex flex-col gap-3"
+                    class="bg-muted p-6 sm:px-9 sm:pt-6 sm:pb-9 rounded flex flex-col gap-3"
```

Comment:
are you sure its bg-muted? i think you want to use bg-default instead of the muted version. We don't use those anywhere

Reply:
Changed it to the default bg indeed.